data/json/items/tool_armor.json

```diff
@@ -1643,7 +1656,8 @@
         "use_action": {
             "type": "transform",
             "msg": "You wrap your scarf a bit tighter.",
-            "target": "scarf_fur"
+            "target": "scarf_fur",
+            "menu_option_text": "Wrap tighter"
```

Comment:
1. What happens if you omit `menu_option_text`?
2. Field name could be shortened to `menu_text`